### ISSUE TITLE
Request for help

### ISSUE THREAD

**Issue by crosshj** [piercus/kalman-filter#59]
Hello @piercus!  Here's a situation where I am trying to leverage this module.  I am happy to PR the solution as an example to share with others who may have a similar issue.

Here's a summary:
1) at a given time, an actor is somewhere on a 1D line, this is represented by Math.sin scaled to range between -10 and +10
2) the actor has access to three noisy sensors
3) based on the input of those sensors and (hopefully) some computations, the actor can guess its position within reason

My engineering coursework got me close to a very basic understanding of Kalman, but not quite there.  I'm hoping a working implementation of the above will push me closer to understanding; this has worked in the past to some degree.

If you would not mind helping, I would be deeply appreciative.

```javascript
const noise = (scale) => (Math.random()-0.5) * scale;

const kalman = (() => {
	const config = {
		//... I am a bit lost here.  Happy to share what I've tried
	};
	const kFilter = new KalmanFilter(config);

	let previousCorrected;
	return (observation /* a 1x3 array */) => {
		previousCorrected = kFilter.filter({previousCorrected, observation});
		return previousCorrected.mean;
	};
})();

let counter = 0;
const step = () => {
	if(counter++ > 10) return;

	const actual = Math.sin(counter)*10;
	const sensor1 = actual + noise(0.25);
	const sensor2 = actual + noise(0.5);
	const sensor3 = actual + noise(0.75);

	const kalmanMean = kalman([sensor1, sensor2, sensor3]);

	// expecting to minimize diff
	const diff = kalmanMean - actual;
	console.log(diff);

	setTimeout(step, 1000);
};

step();
```

**Comment by piercus** [comment]
@crosshj 

The transition could be either very simple, if you only make the assumption that you have a continuous line, or complex if you want to model the sinus of your data inside the kalman filter.

I'm assuming the simple case here, but if you want to make it more complex, you should've read https://dsp.stackexchange.com/a/79460/20012

For a simple transition model, you can state that the modeled variable is continuous by putting

`transition: [[1]]`

Your stateProjection matrix will say that your all 3 sensors are modeling the exact same variable 

`stateProjection: [[1],[1],[1]]` 

I hope it helps :-)

For the covariance just use diagonal matrix with variance in the diag. 

For the init, you can start at 0 with huge covariance.

If it's still not clear, I suggest you re-read the Wikipedia page for kalman-filter, it takes some time to get it, I know, I've been there before 😃

**Comment by piercus** [comment]
@crosshj i took this opportunity to check if extended kalman-filter was effectively working with a sinusoidal function.

And it was not :-) i fixed it in 1.10.0 release and created a [Demo for the sinusoidal Kalman Filter](https://observablehq.com/d/a033acc0859cc0de)

I would be glad to have your feedback on it.
